app/controllers/documents_controller.rb

```diff
@@ -15,18 +15,27 @@ def show
 
   def update
     document = Document.find(params[:id])
-    document.update_attributes(document_update_params(document))
+    base_path = PathGeneratorService.new.path(document, params[:document][:title])
```

Comment:
Could we put this in document_update_params? I'm a fan of making the action methods as small as possible, as they're meant to be the orchestrators.

Reply:
`document_update_params` seems to have a different purpose ie. whitelist the params from the POST to what we expect. I asked about it last week and it seems like it's a rails convention to have these kind of methods of whitelist the POST attributes.

Feels like a separate thing to the path generation.